### ISSUE TITLE
filterscripts: TypeError: 'I3Geometry' object has no attribute '__getitem__' (Trac #2197)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2189]
Hi,

I saw this error while experimenting with the trunk of filterscripts:

```text
Traceback (most recent call last):
  File "./IceCube_BaseProc_only_MM_trunk.py", line 309, in <module>
    tray.Execute()
  File "/data/user/flauber/software/py2-v3.0.1/icerec_trunk/build/lib/I3Tray.py", line 256, in Execute
    super(I3Tray, self).Execute()
  File "/data/user/flauber/software/py2-v3.0.1/icerec_trunk/build/lib/icecube/icetray/i3logging.py", line 28, in log
    record = logging.LogRecord(logger.name, self.pylevels[level], file, line, msg, tuple(), None, None)
  File "/cvmfs/icecube.opensciencegrid.org/py2-v3.0.1/RHEL_7_x86_64/lib/python2.7/logging/__init__.py", line 285, in __init__
    self.threadName = threading.current_thread().name
  File "/cvmfs/icecube.opensciencegrid.org/py2-v3.0.1/RHEL_7_x86_64/lib/python2.7/threading.py", line 1151, in currentThread
    return _active[_get_ident()]
  File "/data/user/flauber/software/py2-v3.0.1/icerec_trunk/build/lib/icecube/icetray/i3logging.py", line 28, in log
    record = logging.LogRecord(logger.name, self.pylevels[level], file, line, msg, tuple(), None, None)
  File "/cvmfs/icecube.opensciencegrid.org/py2-v3.0.1/RHEL_7_x86_64/lib/python2.7/logging/__init__.py", line 285, in __init__
    self.threadName = threading.current_thread().name
  File "/cvmfs/icecube.opensciencegrid.org/py2-v3.0.1/RHEL_7_x86_64/lib/python2.7/threading.py", line 1151, in currentThread
    return _active[_get_ident()]
  File "/data/user/flauber/software/py2-v3.0.1/icerec_trunk/build/lib/icecube/filterscripts/baseproc_superdst.py", line 26, in MaskMaker
    g = omgeo[omkey]
TypeError: 'I3Geometry' object has no attribute '__getitem__'
```

I think the problem is line 26 in 
http://code.icecube.wisc.edu/svn/projects/filterscripts/trunk/python/baseproc_superdst.py

and should be fixed by replacing
``` 
g = omgeo[omkey]
```
with 

```text
g = omgeo.omgeo[omkey]
```
(thanks to alex olivias and jvs for pointing this out to me)
Cheers,
Frederik

Migrated from https://code.icecube.wisc.edu/ticket/2197
```json
{
    "status": "closed", 
    "changetime": "2019-01-12T03:53:01", 
    "description": "Hi,\n\nI saw this error while experimenting with the trunk of filterscripts:\n\n{{{\nTraceback (most recent call last):\n  File \"./IceCube_BaseProc_only_MM_trunk.py\", line 309, in <module>\n    tray.Execute()\n  File \"/data/user/flauber/software/py2-v3.0.1/icerec_trunk/build/lib/I3Tray.py\", line 256, in Execute\n    super(I3Tray, self).Execute()\n  File \"/data/user/flauber/software/py2-v3.0.1/icerec_trunk/build/lib/icecube/icetray/i3logging.py\", line 28, in log\n    record = logging.LogRecord(logger.name, self.pylevels[level], file, line, msg, tuple(), None, None)\n  File \"/cvmfs/icecube.opensciencegrid.org/py2-v3.0.1/RHEL_7_x86_64/lib/python2.7/logging/__init__.py\", line 285, in __init__\n    self.threadName = threading.current_thread().name\n  File \"/cvmfs/icecube.opensciencegrid.org/py2-v3.0.1/RHEL_7_x86_64/lib/python2.7/threading.py\", line 1151, in currentThread\n    return _active[_get_ident()]\n  File \"/data/user/flauber/software/py2-v3.0.1/icerec_trunk/build/lib/icecube/icetray/i3logging.py\", line 28, in log\n    record = logging.LogRecord(logger.name, self.pylevels[level], file, line, msg, tuple(), None, None)\n  File \"/cvmfs/icecube.opensciencegrid.org/py2-v3.0.1/RHEL_7_x86_64/lib/python2.7/logging/__init__.py\", line 285, in __init__\n    self.threadName = threading.current_thread().name\n  File \"/cvmfs/icecube.opensciencegrid.org/py2-v3.0.1/RHEL_7_x86_64/lib/python2.7/threading.py\", line 1151, in currentThread\n    return _active[_get_ident()]\n  File \"/data/user/flauber/software/py2-v3.0.1/icerec_trunk/build/lib/icecube/filterscripts/baseproc_superdst.py\", line 26, in MaskMaker\n    g = omgeo[omkey]\nTypeError: 'I3Geometry' object has no attribute '__getitem__'\n}}}\n\nI think the problem is line 26 in \nhttp://code.icecube.wisc.edu/svn/projects/filterscripts/trunk/python/baseproc_superdst.py\n\nand should be fixed by replacing\n{{{ \ng = omgeo[omkey]\n}}}\nwith \n{{{\ng = omgeo.omgeo[omkey]\n}}}\n(thanks to alex olivias and jvs for pointing this out to me)\nCheers,\nFrederik\n\n\n", 
    "reporter": "flauber", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547265181974434", 
    "component": "infrastructure", 
    "summary": "filterscripts: TypeError: 'I3Geometry' object has no attribute '__getitem__'", 
    "priority": "normal", 
    "keywords": "filterscripts", 
    "time": "2018-10-09T14:06:46", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20181009T16:20:15: 
* olivas changed owner from "" to "olivas"
* olivas changed status from "new" to "closed"
* olivas commented:

> In changeset 166088:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="166088"
> fixed geometry bug introduced in r165893.  closes #2189.
> ```


* olivas changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190112T03:53:01: nwhitehorn commented:

> In changeset 170562:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170562"
> fixed geometry bug introduced in r165893.  closes #2189.
> ```
